stories/core_components/ColumnChart.tsx

```diff
@@ -434,6 +434,26 @@ storiesOf('Core components/ColumnChart', module)
             </div>
         )
     ))
+    .add('optional stacking chart with \'Stack to 100%\' and enabling data labels', () => (
+        screenshotWrap(
```

Comment:
Please make sure to test this for flakiness (running multiple times concurrently) Datalabels have a tendency to be flaky

Reply:
Yes, I'm aware of issue. That's why I only add data labels for column, not bar chart.
According to test cafe result, I think it's ok.